outscale/resource_outscale_load_balancer_policy.go

```diff
@@ -298,6 +312,9 @@ func resourceOutscaleAppCookieStickinessPolicyCreate(d *schema.ResourceData, met
 				a := make(map[string]interface{})
 				a["cookie_name"] = v.CookieName
 				a["policy_name"] = v.PolicyName
+				if v.GetPolicyName() == pn.(string) {
+					d.Set("cookie_name", v.GetCookieName())
+				}
```

Comment:
I don't think this condition is useful

Reply:
It's useful if you have multiples Policies, to update the state with the right name/period